packages/xo-server/src/xo-mixins/xen-servers.js

```diff
@@ -1,6 +1,8 @@
 import createLogger from '@xen-orchestra/log'
+import { BaseError } from 'make-error'
+import { delay as pDelay, ignoreErrors } from 'promise-toolbox'
```

Comment:
Already exposed as `delay` and `pDelay`: https://github.com/JsCommunity/promise-toolbox#imports